bravado/requests_client.py

```diff
@@ -139,7 +141,7 @@ def add_response_detail_to_errors(e):
         args = list(e.args)
         args[0] += (' : ' + e.response.text)
         e.args = tuple(args)
-    raise e
+    raise HTTPError(*e.args), None, sys.exc_info()[2]
```

Comment:
Good point, i needed only when raising back e, but not now. I will add unit tests for expected returned type of e (the input `e` shoud be of type `requests.HTTPError` but not compulsory).